commcare_connect/users/views.py

```diff
@@ -108,11 +108,15 @@ def start_learn_app(request):
         ConnectIDUserLink.objects.create(commcare_username=cc_username, user=request.user, domain=domain)
     try:
         access_object = OpportunityAccess.objects.get(user=request.user, opportunity=opportunity)
+        user_invite = UserInvite.objects.get(opportunity_access=access_object)
```

Comment:
this should be out of the try/catch since the catch isn't relevant to it

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/342/commits/08dacc3d2ab1940cc0e9a4d61c9e2daa7efc0603